Python/initconfig.c

```diff
@@ -63,8 +63,20 @@ static const char usage_3[] = "\
 -W arg : warning control; arg is action:message:category:module:lineno\n\
          also PYTHONWARNINGS=arg\n\
 -x     : skip first line of source, allowing use of non-Unix forms of #!cmd\n\
--X opt : set implementation-specific option. The following options are available:\n\
-\n\
+-X opt : set implementation-specific option (see details with -X help)\n\
+--help-env : print help about other environment variables\n\
```

Comment:
"all" instead of "other"?

Reply:
Other means: other than the ones that you can read about just above in this help output

Reply:
Oh, I see.  Yeah, that's a little weird that `-X help` doesn't actually print *all* the `-X` options.  How hard would that be?

Reply:
`-X help` is fine 😸   Here we are saying that `--help-env` does not print all the environment variables.
It could be done, at the little price of duplicating the info in the `--help` output.

For example, we would have both these lines:
```c
usage_2 = \
"--enable-flufl    : enable flufl to frobnicate the gruzles (also PYTHONENABLEFLUFL=1)\n"

usage_envvars_1 = \
"PYTHONENABLEFLUFL=1  : enable flufl to frobnicate the gruzles (also --enable-fluf)\n"
```

Does that seem good?

Reply:
Oops, yep, I typoed that!

> Does that seem good?

I like it!  You get the fully details either way.  And I say that as an experienced gruzle frobnicator!